### ISSUE TITLE
Translation Requests

### ISSUE THREAD

**Issue by Tech1k** [duino-coin/duino-coin#877]
Hello Duinocoin contributors, we are looking for updated translations for our READMEs and miners!

Please feel free to help contribute to the translation of the Duino-Coin README files here: [https://github.com/revoxhere/duino-coin/tree/master/Resources/README_TRANSLATIONS](https://github.com/revoxhere/duino-coin/tree/master/Resources/README_TRANSLATIONS) and the miners here: [https://github.com/revoxhere/duino-coin/tree/master/Resources](https://github.com/revoxhere/duino-coin/tree/master/Resources) (the _langs.json files)

Any and all help is greatly appreciated and welcomed!

I've made a [GitHub Project](https://github.com/revoxhere/duino-coin/projects/4) to show the progress of translations and show what still needs to be updated!

**Comment by BastelPichi** [comment]
And also, the miner translations could be redone... At least the german one is horrible(because i cant translate lol) and some functions are still in english as theyre new.

**Comment by n0t-f0ur13r** [comment]
Hi, im in, the only other language i know it's spanish (im from Chile so it's my native/mother language), i posted my pull request, i hope is fine.

**Comment by Tech1k** [comment]
> Hi, im in, the only other language i know it's spanish (im from Chile so it's my native/mother language), i posted my pull request, i hope is fine.

I'll review it and thank you for your help!

**Comment by Tech1k** [comment]
> And also, the miner translations could be redone... At least the german one is horrible(because i cant translate lol) and some functions are still in english as theyre new.

I agree with this, I'll update the issue to include the miner translations!

**Comment by Tech1k** [comment]
Updated the title to add more scopes of needed translations!

**Comment by RocketDevil** [comment]
ill help on the German one as much i can Time wise :)

**Comment by Tech1k** [comment]
> ill help on the German one as much i can Time wise :)

Sounds good and any help is appreciated!

**Comment by Vulpecula-nl** [comment]
Im starting now with the Dutch version. I will make a pull request when im finished

Translation completed: see pr #885

**Comment by naphob** [comment]
updated for Thai. check this pr https://github.com/revoxhere/duino-coin/pull/886

**Comment by Tech1k** [comment]
@Vulpecula-nl and @naphob thank you for the PRs, we've merged both of them into the master branch!

**Comment by Tech1k** [comment]
I've created a [GitHub Project](https://github.com/revoxhere/duino-coin/projects/4) to track the progress of translations - at this time I'm still working on setting the project up, but it will tell us what still needs to be updated, etc!

**Comment by Osmodia666** [comment]
Did some error correcting on the German Version.

**Comment by Osmodia666** [comment]
I Don't know why the errors I corrected weren't applied, or why the changes don't even appear anymore in the history, but if you guys don't need or want help, dunno.
There are still plenty of errors in the German translation, I found even more than I corrected at first.

**Comment by RocketDevil** [comment]
> 
> 
> I Don't know why the errors I corrected weren't applied, or why the changes don't even appear anymore in the history, but if you guys don't need or want help, dunno.
> There are still plenty of errors in the German translation, I found even more than I corrected at first.

it looks like u never uploaded your commit file to the master branch, so, therefore, no change in the file was made... 
either upload your commit again or idk.. :D

**Comment by fajarafriansh** [comment]
I'm starting now with the Indonesian version of https://github.com/revoxhere/duino-coin/tree/master/Resources. I will make a pull request after I have finished that.

**Comment by gruiny** [comment]
> And also, the miner translations could be redone... At least the german one is horrible(because i cant translate lol) and some functions are still in english as theyre new.

Hello my friend 

i dont know how to do but i can help with the german part 😅👍
Need only a fast instruktion how to do 😬🙈

**Comment by hotmoist** [comment]
I have worked on creating new Korean version of README file and fixed README.md typo. Please check pr #1093

**Comment by pierrealexaline** [comment]
I can make the work for french next week if you need ? no problem with git and MD format .... Ok i've beginning to work on it ... ready in a few days.
Ok... it's done for the README file.... I'm worked on the CLI_Wallet_langs.json file tomorrow and make the pull request

**Comment by pierrealexaline** [comment]
Can you confirm that you need the french translation only for README.md and CLI_Wallet_langs.json ? thx

**Comment by yutarour** [comment]
Do you need a Japanese translation? If yes, I can do that (although i'm one individual so it might take a while)

**Comment by pierrealexaline** [comment]
ok i've make the pull request from the fork for the french translation have a nice day ! 
Nota : If you need i have made a translation for the "cli_wallet_commands.json" file, i putt it here ...
[cli_wallet_commands_fr_FR.json.zip](https://github.com/revoxhere/duino-coin/files/8190960/cli_wallet_commands_fr_FR.json.zip)

**Comment by Omicron166** [comment]
I did some fixes in README_es_LATAM.md, someone pls merge my pull request #1533

**Comment by MinatoIsuki** [comment]
Hey everyone, I've made a Vietnamese translation, please check my pull request #1650
